cegs_portal/search/templates/search/v1/partials/_glossary.html

```diff
@@ -0,0 +1,207 @@
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    CRISPRi: CRISPR interference, a technique that uses a catalytically dead Cas9 (dCas9) to inhibit gene expression.[<a href="#ref19" class="link">19</a>]
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    CRISPRa: CRISPR activation, a technique that uses dCas9 fused to transcriptional activators to increase gene expression.[<a href="#ref20" class="link">20</a>]
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    Reporter Assay: An assay that uses a reporter gene to measure the activity of a particular gene or regulatory element.[<a href="#ref21" class="link">21</a>]
+                </p>
+                <p class="text-base text-xl leading-relaxed text-gray-800">
+                    Genome Assembly
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    hg-19: GRCh37 (hg19) an earlier human reference genome version. This version has been widely used in genomics projects, including long-term, global studies like the 1,000 Genomes Project.[<a href="#ref22" class="link">22</a>]
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    hg-38: genome assembly GRCh38 (hg38) from Genome Reference Consortium. Corrected and improved version of hg19.[<a href="#ref22" class="link">22</a>]
+                </p>
+                <p class="text-base text-xl leading-relaxed text-gray-800">
+                    Genomic Terms
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    Gene: A gene is the basic physical and functional unit of heredity. Genes are made up of DNA and each chromosome contains many genes.[<a href="#ref23" class="link">23</a>]
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    Transcript: A messenger RNA (mRNA) molecule that carries genetic information from DNA to the cytoplasm, where proteins are made.[<a href="#ref24" class="link">24</a>]
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    Exon: The sections of DNA (or RNA) that code for proteins.[<a href="#ref25" class="link">25</a>]
+                </p>
+                <p class="text-base text-xl leading-relaxed text-gray-800">
+                    Identifiers
+                </p>
+                <p class="text-base leading-relaxed text-gray-600">
+                    Ensembl ID: is an identifier used by the Ensembl database to label features like genes, transcripts, exons, or proteins.[<a href="#ref26" class="link">26</a>]
+                </p>
```

Comment:
Maybe have an entry for "accession id" here too, explaining our accession id system?